compute/openstack.py

```diff
@@ -78,7 +78,7 @@ def get_openstack_driver(
 class CephVMNodeV2:
     """Represent the VMNode required for cephci."""
 
-    default_network_names = [
+    default_rhosd_network_names = [
         "provider_net_cci_12",
         "provider_net_cci_11",
         "provider_net_cci_9",
```

Comment:
Hi Vaibhav,
Is there any restriction to use only this network pools.
if not can we get this dynamically using driver.ex_get_network_pool()

Reply:
@AmarnatReddy if that works, then we can get it dynamically

Reply:
@AmarnatReddy we are redesigning cloud libraries and this includes these openstack operations. This PR is to unblock us for now to use both the OSP projects, Will consider this enhancement as part of redesigning.